### ISSUE TITLE
"new project" has no effect

### ISSUE THREAD

**Issue by buhtz** [Docear/Desktop#494]
I select "new project" in the context menu of the workspace.
The dialog "Create new project" opens and is working. After pressing "Finished" on it the workspace keeps empty. No new project in the workspace.

After closing and reopoen Docear the new project appears.

Using Docear 1.2.0 on Debian GNU/Linux.

**Comment by fulmineo** [comment]
I have the same problem on Windows 7.

**Comment by Joeran** [comment]
restart docear. does the new project appears now?

**Comment by fulmineo** [comment]
Yes! But I thought that it is not normal behavior of the program..

**Comment by kiswanij** [comment]
I have the same issue.
After creating a new project, it will not appear directly in the projects' explorer, docear needs to be restarted in order for the project to appear .
Wonderful project by the way :)

**Comment by chicolucio** [comment]
I have the same problem here. Is the project dead? This is a great project.
